crates/bevy_entropy/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+use bevy_app::{AppBuilder, Plugin};
+use bevy_utils::tracing::{debug, trace};
+use rand::{rngs::StdRng, RngCore, SeedableRng};
+
+pub mod prelude {
+    #[doc(hidden)]
+    pub use crate::Entropy;
+}
+
+/// Provides a source of entropy.
+/// This enables deterministic execution.
```

Comment:
I would definitely qualify this statement some more, to avoid frustrations and confusion.

Reply:
See https://github.com/bevyengine/bevy/discussions/2480 for more issues to be mindful of.